cl/assets/static-global/css/override.css

```diff
@@ -1564,3 +1564,11 @@ textarea {
   overflow-y: auto;
   height: 100vh;
 }
+
+@media (max-width: 780px) {
+    #toc-container {
+        overflow-y: initial;
+        position: relative;
+        height: min-content;
+    }
+}
```

Comment:
Can you explain what this does and maybe post some screenshots, please?

Reply:
This handles the table of contents displaying on top of the other content.  

On mobile we want the table of contents to just work and not be a scrollable list- so we need to stretch it out.

Ill add some photos

Reply:
<img width="596" alt="Screenshot 2023-10-04 at 3 49 14 PM" src="https://github.com/freelawproject/courtlistener/assets/6464529/475f065c-ca15-4b69-8a3e-dd1ace6a951a">

This is attempting to resolve this on mobile 


<img width="593" alt="Screenshot 2023-10-04 at 3 49 29 PM" src="https://github.com/freelawproject/courtlistener/assets/6464529/7f35551e-f89e-4b4b-b089-b106ae3ea418">


This removes the sticky sidebar - and the overlapping text issue and makes it much more usable.